src/workloads/execution/searchScan.yml

```diff
@@ -0,0 +1,98 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  Note that this test requires mongot to run properly, which is currently not supported by genny on evergreen. This test should not be pushed to master.
+
+GlobalDefaults:
+  dbname: &db test
+  MaxPhases: &MaxPhases 20
+  batchSize: &batchSize 30000
+  textField: &field "searchableField"
+  nop: &Nop {Nop: true}
+  searchQuery: &defaultSearch {
+            index: "default", 
+            text: {
+              path: "searchableField", 
+              query: {^FastRandomString: {length: {^RandomInt: {min: 1, max: 3}}, alphabet: "abcd"}},
+              fuzzy: {
+                maxEdits: 2
+              }
+            }
+            }
+
+Clients:
+  Default:
+    URI: "mongodb://__system:keyfile@localhost:27017/local"
+
+Actors:
+#- Name: InsertData
+#  Type: Loader
+#  Threads: 1
+#  Phases:
+#  - Repeat: 1
+#    Database: *db
+#    Threads: 1
+#    CollectionCount: 1
+#    DocumentCount: 100000
+#    BatchSize: *batchSize
+#    Document:
+#      searchableField: {^FastRandomString: {length: {^RandomInt: {min: 1, max: 3}}, alphabet: "abcd"}}
+#      otherFieldFirst: {^RandomInt: {min: 0, max: 10000}}
+#      otherFieldSecond: {^FastRandomString: {length: {^RandomInt: {min: 20, max: 100}}, alphabet: "abcdefghijklmnopqrstuvwxyz"}}
+#  - *Nop
+#  - *Nop
+##  - *Nop
+##  - *Nop
+#
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+  - *Nop
+  - Repeat: 1 
+  - *Nop
+  - *Nop
+
+- Name: SearchScan
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: 3
+    Database: *db
+    Operations:
+    - OperationMetricsName: Search
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline: [
+          {
+            "$search": {
+              index: "default",
+              text: {
+                path: "searchableField",
+                query: "abc",
+                fuzzy: {
+                  maxEdits: 2,
+                  prefixLength: 1
+                }
+              }
+            }
+          }, 
+          {$group: {"_id": null, "count": {"$sum": 1} ,"sum": {"$sum": "$otherFieldFirst"}}}
+        ]
+        cursor: {batchSize: *batchSize}
+  - Repeat: 3
+    Database: *db
+    Operations:
+    - OperationMetricsName: IXScan
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline: [
+          {$match: {searchableField: {"$lt": "b"}}}, 
```

Comment:
Why $lt? That seems somewhat different than the fuzzy search above

Reply:
the keys have at most length of 3, so the fuzzy search of "abc" with 2 edits, and prefix 1 will match everything that starts with "a". So { $lt : "b"} expresses exactly that.